tests/Unit/DataStructures/Tensor/Expressions/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(LIBRARY "Test_Expressions")
 
 set(LIBRARY_SOURCES
   Test_AddSubtract.cpp
+  Test_AddSubSymmetry.cpp
```

Comment:
[nitpick] should reorder this because S < t because ASCII alphabetical order is "ABC...Zabc...z"

Reply:
Whoops! Fixed

Reply:
Even if it wasn't a style guideline, this incorrect ordering bothers me :laughing: